terraform/variables.go

```diff
@@ -1,25 +1,64 @@
 package terraform
 
 import (
+	"strings"
+
 	"github.com/hashicorp/hcl/v2"
 	"github.com/zclconf/go-cty/cty"
 )
 
-func extractFromFile(file *hcl.File) (VariableMap, hcl.Diagnostics) {
-	// TODO: remove temporary dummy variables
-	// implement "var" when we do input variables and default values
+type VariableMap map[string]cty.Value
+
+func extractFromFile(filename string, file *hcl.File) (VariableMap, hcl.Diagnostics) {
+	// TODO: remove temporary dummy locals
 	// implement "local" when we do local values
+	varMap := VariableMap{}
+	var hclDiags hcl.Diagnostics
+
+	if strings.HasSuffix(filename, TF) {
```

Comment:
>if there was another reason you were thinking about doing the check here, happy to hear, I might be also missing something I haven't thought of :D

The reason we did this check here is because `.tfvars` files will have a different logic to extract vars from them.
If this condition wasn't here than the logic will fail when we will try to extract vars from `.tfvars` files.

Reply:
please correct me if I'm wrong (I joined in the middle so I might missed some stuff), but we are sending to the `ParseModule` function also `.tfvars` files no? AFAIK they are part of a TF module.
If we do than we need this condition to "filter" out the `.tfvars` files.

Reply:
okay I understand, in this case it does make sense to remove this condition.